bool isZoom
bool isPan
bool isTilt
float32 zoom
float32 pan
float32 tilt